.github/ISSUE_TEMPLATE/questions-help-support.md

```diff
@@ -13,8 +13,8 @@ assignees: ''
 
 If you are more comfortable with [Stack Overflow], you may consider posting your question there instead.
 Alternatively, for issues that would benefit from more of an interactive session with the developers,
-you may refer to the `#general` channel in the [chainer slack] or the [chainer-jp slack] (Japanese) Slack workspaces.
+you may refer to the [optuna/optuna chat] or the [optuna/optuna-jp chat] (Japanese) on Gitter.
```

Comment:
Sorry for this comment after LGTM.
I agree with @hvy's opinion (https://github.com/optuna/optuna/pull/750#issuecomment-559745950). How about removing the link to `optuna/optuna-jp` room from here?

```suggestion
you may refer to the [optuna/optuna chat] on Gitter.
```

Reply:
Thank you for sharing your opinion. As mentioned [here](https://github.com/optuna/optuna/pull/750#issuecomment-560202428), I'll remove it.

In addition to your suggested change, I'll also remove the URL. So, please wait for a while.

Reply:
Fixed by https://github.com/optuna/optuna/pull/750/commits/3eb7392e7990bad47f6959063cf2928173903190 and https://github.com/optuna/optuna/pull/750/commits/23752809f679cfe0407ad6780a4a350c6158101c.